cnew.py

```diff
@@ -59,7 +59,7 @@ async def chk_username_presence(web_socket, user_name, chat_room):
 
 async def hello(servaddr, username, chatroom, password):
     async with websockets.connect(servaddr) as websocket:
-        chkUserPresense = await chk_username_presence(websocket,username, chatroom)
+        chkUserPresence = await chk_username_presence(websocket,username, chatroom)
```

Comment:
The server-end reports tons of errors on the occassion of user joining/leaving and chat transfers.

![image](https://user-images.githubusercontent.com/49605954/96328417-673eb600-1060-11eb-9c2f-016614e3ac58.png)

Take a look.

Reply:
I'll be back with the corrections.

Reply:
I have resolved the above issue